vincentmuriuki/API_3
# #162038598 Feature to `GET` all customer orders

## What does this PR do?
- It adds the `GET` endpoint used for retrieving all customer orders
## How to test the endpoint
- git clone 'git@github.com:vincentmuriuki/API_3.git'
- Navigate to `API_3` directory
- Create a virtual environment: `virtualenv venv`
- Then activate the virtual environment: 
                  Windows: `source venv/Scripts/activate`
                  Linux: `source/bin/activate`
- Run `pip install -r requirements.txt`
- Then run `python run.py`
- Open `Postman` and add the following url `http://127.0.0.1:5000/api/v1/orders`
- Ensure you use the `GET` request then hit `Enter`

You will get a result such as the one below:

## Screenshots
![screenshot from 2018-11-18 20-15-55](https://user-images.githubusercontent.com/18227536/48710900-ce585380-ec1a-11e8-80a2-6c11a3c3eac3.png)
